app/src/main/AndroidManifest.xml

```diff
@@ -271,6 +271,9 @@
     <activity
       android:name=".app.testing.activity.TestActivity"
       android:theme="@style/OppiaThemeWithoutActionBar" />
+    <activity
+      android:name=".app.testing.FractionInputInteractionViewTestActivity"
+      android:theme="@style/OppiaThemeWithoutActionBar" />
```

Comment:
```xml
<activity android:name=".app.testing.FractionInputInteractionViewTestActivity" />
```

Reply:
Also, keep it below the line -

```xml
<activity android:name=".app.testing.InputInteractionViewTestActivity" />
```
Good for readability.

Reply:
done